Adjust the lid Parameter on the Rate Limited Flapper - May 16, 2022.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Adjust the Lid Parameter on the Rate Limited Flapper, May 16, 2022
+summary: Signal your support or opposition to Adjust the Lid Parameter on the Rate Limited Flapper
+discussion_link: https://forum.makerdao.com/t/signal-request-change-lid-parameter-on-the-rate-limiter/14688
+vote_type: Plurality Voting
+categories:
+   - Risk Variable
+   - Technical
+   - Auctions
+   - Medium Impact
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-05-16T:00:00
+end_date: 2022-05-19T:00:00
+---
+# Poll:Adjust the lid Parameter on the Rate Limited Flapper - May 16, 2022
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the MakerDAO Community. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, May 16 at 16:00 UTC.
+
+**This is a plurality vote.** 
+- **You may vote for a single option.**
+- **You should vote for the option that you prefer.** 
+- **If you would accept any of the options you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following changes: 
+
+* Adjust the lid parameter from 150,000 DAI to 30,000 DAI limiting the amount of parallel flap auctions to one.
+
+Please review the discussion [thread](https://forum.makerdao.com/t/signal-request-change-lid-parameter-on-the-rate-limiter/14688) to help inform your position before voting.
+
+For additional information on the parameters please click [here](https://forum.makerdao.com/t/rate-limited-flapper/13056) and [here](https://forum.makerdao.com/t/rate-limiter-parameter-proposal/13193) .
+
+## Outcomes
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following actions will be taken:**
+* This change will be included in an upcoming Executive Vote as the Protocol Engineering Core Unit's schedule allows. 
+* It is expected that this Executive Vote will take place within 30 days of this poll passing, absent external factors.
+* If the Friday executive passes, then this change will become active in the Maker Protocol after the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay) has expired.
```

Comment:
"If the executive passes" - execs aren't on Friday's anymore

I'll change the template for this too